services/project-grafana-loki/0.48.5/object-bucket-claims.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: project-loki-object-bucket-claims
+  namespace: ${releaseNamespace}
+spec:
+  chart:
+    spec:
+      chart: object-bucket-claim
+      sourceRef:
+        kind: HelmRepository
+        name: mesosphere.github.io-charts-stable
+        namespace: kommander-flux
+      version: 0.1.1
+  interval: 15s
+  install:
+    crds: CreateReplace
+    remediation:
+      retries: 30
+  upgrade:
+    crds: CreateReplace
+    remediation:
+      retries: 30
+  releaseName: project-loki-object-bucket-claims
+  valuesFrom:
+    - kind: ConfigMap
+      name: project-grafana-loki-0.48.5-d2iq-defaults
```

Comment:
We’ll need to manually include both the workspace overrides and the cluster overrides cm names here
https://github.com/mesosphere/kommander-applications/pull/685/files#r992710850

Reply:
added WS override but skipped the cluster override since we don support cluster scope in project.

Reply:
Oo good catch